config.selenium-standalone-chrome.yaml

```diff
@@ -4,6 +4,16 @@
 #
 # This file comes from https://github.com/drud/ddev-selenium-standalone-chrome
 #
+hooks:
+  # Make sure that the standard mkcert CA keys are readable
+  # as they normally are owned by the ddev-defined user, and here we're running
+  # as 'seluser'
+  # and do a mkcert -install so we can trust them
+  post-start:
+    - exec: "sudo chmod ugo+r /mnt/ddev-global-cache/mkcert/*"
+    - exec: "mkcert -install"
+      service: selenium-chrome
```

Comment:
And here we do the mkcert -install inside the selenium-chrome container. This can't be done earlier (like in Dockerfile) because we have to have access to the CA, mounted in /mnt/ddev-global-cache/mkcert